### PR TITLE
Doc Fares: added specs for writing NTFS fares

### PR DESCRIPTION
**Pull request by papailio**
First version of specs for writing NTFS fares (provided that at this stage, the NTFS fares model is far richer than the internal fares model).

### CODE REVIEW

File: src/documentation/write_ntfs_fares.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+In this initial version: 
+- only the files [prices.csv](#pricescsv) and [od_fares.csv](#od_farescsv) are written, as the NTM fares model does not yet support special conditions for the tickets
+- only tickets on origin-destination stops are taken into account.
+- tickets specified in a currency different than EUR are ignored.
+
+These limitations will be adressed in a later version.
+
+### prices.csv
+As a reminder, this file has no header and the order of the NTFS fields must be respected.
+NTFS field | NTM object | NTM property | Notes/Mapping rule
```
Comment (pbougue) [inline src/documentation/write_ntfs_fares.md]: You probably miss a blank line here for layout.

File: src/documentation/write_ntfs_fares.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Writing NTFS fares
+## Introduction
+This document describes how fares specified in Navitia Transit Model are transformed into a [NTFS fare feed](https://github.com/CanalTP/navitia/blob/dev/documentation/ntfs/ntfs_fare_extension_fr.md).
+
+In this initial version: 
+- only the files [prices.csv](#pricescsv) and [od_fares.csv](#od_farescsv) are written, as the NTM fares model does not yet support special conditions for the tickets
```
Comment (pbougue) [inline src/documentation/write_ntfs_fares.md]: Suggestion: To be verified but I think od_fares.csv is fully included in the model of fares.csv .
Can it be relevant to just keep the last one? Not fan of having multiple ways to express the same thing in a model, so I'd choose the larger part and progressively depreciate the restricted (yet simpler) od_fares.csv .

File: src/documentation/write_ntfs_fares.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+These limitations will be adressed in a later version.
+
+### prices.csv
+As a reminder, this file has no header and the order of the NTFS fields must be respected.
+NTFS field | NTM object | NTM property | Notes/Mapping rule
+--- | --- | --- | ---
+\*clef de ticket\* | Ticket | ticket_id
+\*date de début de validité\* | Ticket | start_date | Starting date of the validity period of the fare structure in the form YYYYMMDD.
+\*date de fin de validité\* | Ticket | end_date | The date after the specified end date in the form YYYYMMDD.
+\*prix\* | Ticket | price | The specified value is converted into euro cents.
```
Comment (pbougue) [inline src/documentation/write_ntfs_fares.md]: I'd add : is converted into euro cents `in NTFS field` for clarity.
But in NTM field, it's already integer cents, no? (At least that's what I understood from using the norm).

File: src/documentation/write_ntfs_fares.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+\*name\* | | | Fixed value `Ticket Orgine-Destination`.
+\*champ ignoré\* |  |  | This field is explicitly left empty.
+\*commentaire\* | | | This field is explicitly left empty.
+\*devise\* | Ticket | currency_type | The value is set to `centime` provided that the currency used for the ticket is EUR.
```
Comment (pbougue) [inline src/documentation/write_ntfs_fares.md]: This is not a hard rule for navitia2, it mostly depends on existing integrations.
It's just like that everywhere, but fronts should probably adapt at some point.

File: src/documentation/write_ntfs_fares.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+As a reminder, this file has no header and the order of the NTFS fields must be respected.
+NTFS field | NTM object | NTM property | Notes/Mapping rule
+--- | --- | --- | ---
+\*clef de ticket\* | Ticket | ticket_id
+\*date de début de validité\* | Ticket | start_date | Starting date of the validity period of the fare structure in the form YYYYMMDD.
+\*date de fin de validité\* | Ticket | end_date | The date after the specified end date in the form YYYYMMDD.
+\*prix\* | Ticket | price | The specified value is converted into euro cents.
+\*name\* | | | Fixed value `Ticket Orgine-Destination`.
+\*champ ignoré\* |  |  | This field is explicitly left empty.
+\*commentaire\* | | | This field is explicitly left empty.
```
Comment (pbougue) [inline src/documentation/write_ntfs_fares.md]: minor: not the same spacing

File: src/documentation/write_ntfs_fares.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+\*devise\* | Ticket | currency_type | The value is set to `centime` provided that the currency used for the ticket is EUR.
+
+### od_fares.csv
+
+NTFS field | NTM object | NTM property | Notes/Mapping rule
+--- | --- | --- | ---
+Origin ID | OD Rules | origin_stoparea_id | 
+Origin name | | This field is explicitly left empty.
```
Comment (pbougue) [inline src/documentation/write_ntfs_fares.md]: Comment is not on the same column as previous table.

File: src/documentation/write_ntfs_fares.md (hunk 2 above)
Comment (papailio) [reply src/documentation/write_ntfs_fares.md]: I'd rather leave it as is for now, we need to keep read/write as simple as possible given the specificities of the input data (netex_nl). I note the remark though, as the internal model gets richer, the writing will be updated as well.

**Pull request merged**